### ISSUE TITLE
Updated to new version Coffe icon

### ISSUE THREAD

**Issue by upup666** [mangerlahn/Latest#302]
What means coffee icon near apps, after new update i've got a lot of apps that lists not updated, but I need to open each one to update them, also some of them not updating at all.

**Comment by hbre** [comment]
I think it's not coffee ☕ but beer 🍺 and refers to software installed via [homebrew](https://brew.sh). Just running `brew update` and `brew upgrade` on my machine

**Comment by upup666** [comment]
Yep its really a beer icon) but brew update and upgrade doesn't work; (

**Comment by mangerlahn** [comment]
Dupe of #337
